yapapi/log.py

```diff
@@ -64,6 +64,17 @@
 _agreements_pool_logger = logging.getLogger("yapapi.agreements_pool")
 
 
+class _YagnaDatetimeFormatter(logging.Formatter):
+    """Custom log Formatter that formats datetime using the same convention yagna uses."""
+
+    LOCAL_TZ = datetime.now(timezone.utc).astimezone().tzinfo
+
+    def formatTime(self, record: logging.LogRecord, datefmt=None):
+        """Format datetime; example: `2021-06-11T14:55:43.156.123+0200`."""
+        dt = datetime.fromtimestamp(record.created, tz=self.LOCAL_TZ)
+        return dt.strftime(f"%Y-%m-%dT%H:%M:%S.{dt.microsecond / 1000}%z")
```

Comment:
```suggestion
        """Format datetime; example: `2021-06-11T14:55:43.156+0200`."""
        dt = datetime.fromtimestamp(record.created, tz=self.LOCAL_TZ)
        return dt.strftime(f"%Y-%m-%dT%H:%M:%S.{int(dt.microsecond / 1000)}%z")
```

Reply:
Thanks, @filipgolem, good catch! And I need also to remember about the leading 0's, in case the number of milliseconds is less than 100.